### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by rutu0312**

### CODE REVIEW

File: src/Dashboard/Event.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+function Event() {
+
+    const [searchTerm, setSearchTerm] = useState("")
+    const [dataaa, setDataaa] = useState()
+    useEffect(() => {
+        try {
+            listOfLatestEventApi().then(response => setDataaa(response?.data))
+        } catch (error) {
+
```
Comment (pooja-vipulbhai-darji) [inline src/Dashboard/Event.jsx:16]: what about the catch

File: src/Dashboard/Event.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        event_description: event.event_description,
+        event_manage_images: event.event_manage_images,
+        created_at: event.created_at,
+    }))
+
+    const filteredEvents = processedData?.filter((event) => {
+        const lowerSearchTerm = searchTerm.toLowerCase();
+        return (
+            String(event.id)?.toLowerCase().includes(lowerSearchTerm) ||
+            event.event_name?.toLowerCase().includes(lowerSearchTerm) ||
```
Comment (pooja-vipulbhai-darji) [inline src/Dashboard/Event.jsx:32]: what is this ? elaborate with me

File: src/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -1,11 +1,197 @@
...
+            countOfTotalUserApi().then(response => setDataa(response?.data))
+            bookingReportApi().then(response => setBooking(response?.data))
+            graphOfUserApi().then(response => setGraphData(response?.data))
+        } catch (error) {
+
+        }
+    }
+        , [])
+
+    console.log(graphData)
```
Comment (pooja-vipulbhai-darji) [inline src/Dashboard/Dashboard.jsx:30]: remove this console

File: src/Dashboard/Dashboard.jsx [hunk 4]
```diff
@@ -1,11 +1,197 @@
...
+        email: event.auth_user.email,
+        phone_number: event.auth_user.phone_number,
+        status: event.status,
+    }))
+
+    const filteredEvents = processedData?.filter((event) => {
+        const lowerSearchTerm = searchTerm.toLowerCase();
+        return (
+            String(event.id)?.toLowerCase().includes(lowerSearchTerm) ||
+            event.auth_user.name?.toLowerCase().includes(lowerSearchTerm) ||
```
Comment (pooja-vipulbhai-darji) [inline src/Dashboard/Dashboard.jsx:43]: elaborate with me

File: src/Dashboard/Dashboard.jsx [hunk 5]
```diff
@@ -1,11 +1,197 @@
...
+                                        </div>
+                                    </div>
+                                </div>
+                                <div className="col-6">
+                                    <div className="card radius-10 border-start border-4 border-5  border-warning">
+                                        <div className="card-body">
+                                            <div className="d-flex align-items-center">
+                                                <div>
+                                                    <p className="mb-0 text-secondary">{dashboardLables.pendingBooking}</p>
+                                                    <h4 className="my-1 text-warning">{data?.[0]?.countOfPendingBooking}</h4>
```
Comment (pooja-vipulbhai-darji) [inline src/Dashboard/Dashboard.jsx:105]: remove this hard code from all files

File: src/Dashboard/Event.jsx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+                        <hr className='mt-2' />
+                        <div className="card">
+                            <div className="card-body ">
+                                <div className="table-responsive  ">
+                                    <BaseTable tableName='Event register' columns={columns} onSearch={handleSearch} searchTerm={searchTerm} Data={filteredEvents?.map((value) => ({
+                                        ...value,
+                                        event_manage_images: (
+                                            <img
+                                                src={`${imageURL}` / `${value.event_manage_images?.[0]?.image}`}
+                                                style={{ height: "50px", width: "80px", objectFit: "cover" }}
```
Comment (pooja-vipulbhai-darji) [inline src/Dashboard/Event.jsx:74]: don't give the inline style please remove from whole file

File: src/Dashboard/Event.jsx (hunk 2 above)
Comment (rutu0312) [reply src/Dashboard/Event.jsx:32]: This code converts the search term to lowercase and checks if it matches either the lowercase ID or lowercase event name of an event object, with optional chaining to handle null/undefined properties.

File: src/Dashboard/Dashboard.jsx (hunk 4 above)
Comment (rutu0312) [reply src/Dashboard/Dashboard.jsx:43]: This code converts the search term to lowercase and checks if it matches either the lowercase ID or lowercase name of the authenticated user associated with an event object, handling null/undefined properties.

**Pull request merged**